### PR TITLE
Update documentation page template v2

### PR DESCRIPTION
**Pull request by maya**
It will look slightly out of sync until header navigation is updated to v2.

[😎 Preview](https://federalist-proxy.app.cloud.gov/preview/uswds/uswds/update-docs-layout/components/preview/layout--docs.html)

### CODE REVIEW

**Comment by maya** [comment]
This is not done, but would like a review of the strategy, cc @thisisdano

**Comment by maya** [comment]
On hold til sidenav is complete.

File: src/stylesheets/components/_layout.scss [hunk 1]
```diff
@@ -1,36 +1,15 @@
...
-}
-
 .usa-layout-docs-sidenav {
   order: 2;
-}
-
-.usa-layout-docs-main_content {
-  margin-bottom: $site-margins;
-  order: 1;
+  padding-top: units(6);
```
Comment (maya) [inline src/stylesheets/components/_layout.scss]: @thisisdano how much space should we have between the bottom of the page and the nav below?

Comment (thisisdano) [reply src/stylesheets/components/_layout.scss]: 4 units seems like a good place to start

Comment (maya) [reply src/stylesheets/components/_layout.scss]: Updated in 6e52d90, thanks! Also, this PR is ready to review 👓

**Pull request merged**